README.md

```diff
@@ -3,3 +3,12 @@
 
 
 This is the codebase for the Colligo application, showcasing digital manuscripts.
+
+## Local Development Setup
+Run `bundle exec rake colligo:install` to install gems, jetty, migrate the DB, and start the Solr server with fixtures.
```

Comment:
This rake task doesn't start the Solr server for me -- I had to do that manually and run `colligo:install` twice...

```
$ bundle exec rake colligo:install
Downloading jetty
Unzipping jetty
cp /Users/drh/workspace/colligo/config/solr_configs/schema.xml /Users/drh/workspace/colligo/jetty/solr/blacklight-core/conf/
cp /Users/drh/workspace/colligo/config/solr_configs/solrconfig.xml /Users/drh/workspace/colligo/jetty/solr/blacklight-core/conf/
rake aborted!
Errno::ECONNREFUSED: Connection refused
```

in other words, this worked...

```
$ bundle exec rake colligo:install # get connection refused error
$ bundle exec rake jetty:start colligo:install
```